ClosedXML/Excel/EnumConverter.cs

```diff
@@ -780,6 +781,45 @@ public static Xdr.EditAsValues ToOpenXml(this XLPicturePlacement value)
             }
         }
 
+        public static OfficeExcel.SparklineTypeValues ToOpenXml(this XLSparklineType value)
+        {
+            switch (value)
+            {                
+                case XLSparklineType.Line: return OfficeExcel.SparklineTypeValues.Line;
+                case XLSparklineType.Column: return OfficeExcel.SparklineTypeValues.Column;
+                case XLSparklineType.Stacked: return OfficeExcel.SparklineTypeValues.Stacked;
+
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(value), "Not implemented value!");
+            }
+        }
+
+        public static OfficeExcel.SparklineAxisMinMaxValues ToOpenXml(this XLSparklineAxisMinMax value)
+        {
+            switch (value)
+            {
+                case XLSparklineAxisMinMax.Individual: return OfficeExcel.SparklineAxisMinMaxValues.Individual;
+                case XLSparklineAxisMinMax.Group: return OfficeExcel.SparklineAxisMinMaxValues.Group;
+                case XLSparklineAxisMinMax.Custom: return OfficeExcel.SparklineAxisMinMaxValues.Custom;
+
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(value), "Not implemented value!");
```

Comment:
For the enum conversions  I was following the pattern that the other enums were using. It seems like it would this be better to change this when all of the others are changed as well?

Reply:
Fair point, I missed this. Let's keep in line with other enums